### ISSUE TITLE
[BUG]Deprecated tag is generated at config exporter

### ISSUE THREAD

**Issue by kazogihara** [ProgrammableMatterProject/VisibleSim#12]
Hi!
I found deprecated config tag is generated at config generator in context menu.
I think this needs to be fixed, could you please check it?

* generated config example

```
<?xml version="1.0" standalone="no" ?>
<world gridSize="5,5,6" windowSize="1280,720">
    <camera target="25,20,10" directionSpherical="0,30,100" angle="30.000000" near="1.000000" far="300.000000" />
    <spotlight target="25,25,0" directionSpherical="-35,40,150" angle="45.000000" />
    <blockList blockSize="10,10,10">
        <block position="0,0,0" color="128,128,64" orientation="0" />
        <block position="1,0,0" color="128,128,64" orientation="0" />
    </blockList>
</world>
```

**Comment by docben** [comment]
Yes the new schema is the following, I'm currently working on this issue.

<?xml version="1.0" standalone="no" ?>
<vs>
<visuals>
	<window size="1280,720" backgroundColor="0xffffff" gradientColor="0x00ffff"/>
	<render shadows="true" grid="true"/>
</visuals>
<world gridSize="5,5,6">
	<camera target="25,20,10" thetaPhiDist="0,30,100" fov="30" near="1" far="300" />
	<spotlight target="25,25,0" thetaPhiDist="-35,40,150" fov="45"/>

	<blockList color="GREY" orientation="0">
            <block position="0,0,0" color="128,128,64" orientation="0" />
            <block position="1,0,0" color="128,128,64" orientation="0" />
	</blockList>
</world>
</vs>
